content/java/concepts/math-methods/terms/max/max.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.max()'
+Description: 'Returns the maximum value from the given two arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Function'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.max()`** method returns the maximum value from the given two arguments.
+
+## Syntax
+
+```pseudo
+Math.max(num1, num2)
+```
+- The `.max()` function takes two parameters `num1` and `num2`.
+- The `num1` and `num2` arguments can be of type int, float, double or long.
```

Comment:
link type to: https://www.codecademy.com/resources/docs/java/data-types

Reply:
Also, maybe let's place the types in backticks. Can we also do this for the other entry

Reply:
Hey @yangc95 
I didn't understand what I have to do in this, do I have to make "type" word clickable which will redirect to https://www.codecademy.com/resources/docs/java/data-types ?

Reply:
Hi @MinalGhate,

Yes! Exactly~